src/gui/toolbarMenubar/ToolMenuHandler.cpp

```diff
@@ -309,12 +300,18 @@ void ToolMenuHandler::signalConnectCallback(GtkBuilder* builder, GObject* object
     addToolItem(new ToolButton(listener, name, action, stockIcon, text))
 
 // Use Custom loading Icon
-#define ADD_CUSTOM_ITEM(name, action, icon, text) addToolItem(new ToolButton(listener, name, action, icon, text))
+#define ADD_CUSTOM_ITEM(name, action, icon, text) \
```

Comment:
Maybe also a lambda for the following `#define`s

Reply:
Done Please check!

Reply:
LGTM.